test/models/project_media_8_test.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../test_helper'
+
+class ProjectMedia8Test < ActiveSupport::TestCase
+  def setup
+    super
+    require 'sidekiq/testing'
+  end
+
+  test "when creating an item with tag/tags, tags should be created in the background" do
+    Sidekiq::Testing.inline!
+
+    t = create_team
+    p = create_project team: t
+    assert_nothing_raised do
+      create_project_media project: p, tags: ['one']
+    end
```

Comment:
This test should be something like this to test background jobs
t = create_team
Sidekiq::Testing.fake! do
 pm = create_project_media team: t, tags: ['one']
assert_difference 'Tag.count' do
 Sidekiq::Worker.drain_all
end
 assert_equal 1, pm.annoations('tag').count
end

Reply:
@melsawy a question for when you are back:
Is there a difference between using `Sidekiq::Testing.inline!` vs `Sidekiq::Testing.fake!` \  `Sidekiq::Worker.drain_all`?

I updated it like this:
https://github.com/meedan/check-api/blob/353612139eca53830ed062ac8612cc03b36a9a11/test/models/project_media_8_test.rb#L27-35

Reply:
when we need to make sure that job created in background we should use `Sidekiq::Testing.fake!` and check the jobs size otherwise the both are same.